src/resource-metadata/template.yaml

```diff
@@ -115,6 +119,10 @@ Parameters:
     Type: String
     Description: '[True/False]'
     Default: 'False'
+  ResourceTypeExclude:
```

Comment:
Both in JS code and in the CF templace we convert this "ec2" / "lambda" string into two booleans. Why not have the user input the booleans (IsEC2ResourceTypeExcluded, IsLambdaResourceTypeExcluded) directly? That would also make extending it with a third type of resource easier

Reply:
okay, changed to boolean keyval